logger.log

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE log SYSTEM "logger.dtd">
+<log>
+<record>
+  <date>2024-03-23T14:57:47.232114Z</date>
+  <millis>1711205867232</millis>
+  <nanos>114000</nanos>
+  <sequence>0</sequence>
+  <logger>RecipeIO Logger</logger>
+  <level>INFO</level>
+  <class>recipeio.RecipeIO</class>
+  <method>runCommandLoopUntilExitCommand</method>
+  <thread>1</thread>
+  <message>asking for first input from user.</message>
+</record>
+<record>
+  <date>2024-03-23T14:57:53.138028Z</date>
+  <millis>1711205873138</millis>
+  <nanos>28000</nanos>
+  <sequence>1</sequence>
+  <logger>RecipeIO Logger</logger>
+  <level>INFO</level>
+  <class>recipeio.RecipeIO</class>
+  <method>runCommandLoopUntilExitCommand</method>
+  <thread>1</thread>
+  <message>not an exit command.</message>
+</record>
+<record>
+  <date>2024-03-23T14:57:55.129339Z</date>
+  <millis>1711205875129</millis>
+  <nanos>339000</nanos>
+  <sequence>2</sequence>
+  <logger>RecipeIO Logger</logger>
+  <level>INFO</level>
+  <class>recipeio.RecipeIO</class>
+  <method>runCommandLoopUntilExitCommand</method>
+  <thread>1</thread>
+  <message>not an exit command.</message>
+</record>
```

Comment:
could you include the .log file in the gitignore file? I don't think we need to push the .log file onto github.